src/kernel/src/process/module/recompiler/x64.rs

```diff
@@ -1757,9 +1911,29 @@ impl Recompiler for X64Emitter<'_> {
     fn transform_vmovups_ymm_ymmm256(&mut self, i: Instruction) -> usize {
         // Check if second operand use RIP-relative.
         if i.op1_kind() == OpKind::Memory && i.is_ip_rel_memory_operand() {
-            panic!(
-                "VMOVUPS ymm1, ymm2/m256 with second operand as RIP-relative is not supported yet."
-            );
+            let dst = i.op0_register();
+            let src = i.ip_rel_memory_address();
+            let tmp = get_gpr64(Self::temp_register64(dst)).unwrap();
```

Comment:
We don't need this in this case because the destination operand is `ymm`, which mean we can use any general purpose register (e.g. `rax`) without conflicting.